supabase/README.md

```diff
@@ -0,0 +1,3 @@
+# Supabase Files
+
+This directory contains the files for the Supabase project. They do not really belong in this repository, but they are here for convenience. Whenever you make changes to these files, you should also make the same changes in the Supabase instance (and vice versa).
```

Comment:
I wouldn't say they don't belong in this repository, as they are inseparably linked to the project. At the very least, I think this will contain a great amount of code that will essentially act as documentation we can reference in our other code, e.g. supabase functions. I know you mean the code isn't actually called from anywhere, but please formulate it less negatively :)

Reply:
Yes good point, I wanted to reformulate that.